Added steps for hybrid scenario (#2304)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/collaboration-exo/public-folders/assign-permissions-mail-enabled-pfs.md

- [Set-MailPublicFolder](https://docs.microsoft.com/powershell/module/exchange/set-mailpublicfolder)

- [Add-RecipientPermission](https://docs.microsoft.com/powershell/module/exchange/add-recipientpermission)

> ## Send As mail enabled public folder in Hybrid scenario

Reviewer: ```suggestion
## Send As mail-enabled public folder in a hybrid scenario
```

> For Exchange Online mailboxes accessing public folders deployed at On-Premises:

Reviewer: ```suggestion
For Exchange Online mailboxes accessing public folders deployed on-premises:
```

> 1) Ensure Mail Enabled Public Folders are synced to Exchange Online 

Reviewer: ```suggestion
1) Ensure that mail-enabled public folders are synced to Exchange Online.
```

> Use following EXO PowerShell command to ensure On-Premises mail public folder are synced:

Reviewer: ```suggestion
Use the following EXO PowerShell command to ensure that on-premises mail public folders are synced:
```

`Get-MailPublicFolder <MEPFName>`

> Example:

Reviewer: ```suggestion
Example
```

> Example:

Reviewer: ```suggestion
```

> Following example lists MEPF named OnPremPF

Reviewer: ```suggestion
The following example lists mail-enabled public folders named OnPremPF:
```

`Get-MailPublicFolder OnPremPF`

> If the MEPF from On-Premises are not showing in EXO, use the Sync-MailPublicFolders.ps1 (for Exchange Server 2010) or Sync-ModernMailPublicFolders.ps1 (For Exchange 2013/2016/2019) to sync the MEPF's first.

Reviewer: ```suggestion
If the on-premises mail-enabled public folders are not displayed, use Sync-MailPublicFolders.ps1 (for Exchange Server 2010) or Sync-ModernMailPublicFolders.ps1 (for Exchange 2013/2016/2019) to sync the mail-enabled public folders first.
```

> 2) Use following command in EXO PowerShell to assign SendAs permission:

Reviewer: ```suggestion
2) Use the following command in EXO PowerShell to assign SendAs permission:
```
